### PR TITLE
Re-Open sentience event

### PR DESCRIPTION
**Pull request by Aurorablade**
orginally in https://github.com/ParadiseSS13/Paradise/pull/7453

Simple mobs that have the SENTIENCE_ORGANIC flag will be elible for a sentience event. allowing them to talk and understand people.

:cl: Purpose2 and Re-Opened by Fethas
rscadd: Rare Sentience event, the mice now want coffee!
/:cl:

### CODE REVIEW

**Comment by SamHPurp** [comment]
Could you update the byline to "Purpose2" rather than Sam if you must include me ;)

**Comment by Aurorablade** [comment]
my apologies!

**Comment by taukausanake** [comment]
Does this still only make a single random simple mob on the station sentient? Really feel it's lackluster if only that one but possibly breaking if too many.
Offtopic: Part of me wishes sentient potions were a liquid you could grenade with just to see how the ghost community would take it. Might also break something but curiosity as it is

**Comment by Aurorablade** [comment]
single simple_animal. give them the ability to speak to others. but it can be anything on the z-level thats a simple animal. We could up it to monkeys but our monkey system is strange.

**Comment by Aurorablade** [comment]
i actually had another idea for them but dunno if  should do it...i could add fluff objectives to it. nothing harmful..

**Comment by SamHPurp** [comment]
As per the previous instructions of add the bare minimum then scale up, I'd say don't add them.

**Comment by taukausanake** [comment]
Adding the bare minimums almost kept the Coroner away as a normal job. Kind of shitty process for features if you ask me.

Thinking a little more: what else would make this mob special besides just giving it the power to talk? It sounds only like a gimmick right now and that's not bad but it's kind of boring too

**Comment by SamHPurp** [comment]
@taukausanake thats ultimately why I closed my original PR ;)

**Comment by Aurorablade** [comment]
i can give it fluff objectives as i said.

**Comment by alexkar598** [comment]
Pull the same pair of shoes on 50 different tiles.
Escape alive on the shuttle

**Comment by Aurorablade** [comment]
I will add fluff objectives later

File: code/modules/events/sentience.dm [hunk 1]
```diff
@@ -26,9 +26,43 @@
 	SA.health = SA.maxHealth
 	SA.del_on_death = FALSE
 	greet_sentient(SA)
+	var/objtype = pick(subtypesof(/datum/objective/sentFluff/))
+	var/datum/objective/sentFluff/O = new objtype()
+	SA.mind.objectives += O
+	to_chat(SA, "<B>Purpose in life</B>: [O.explanation_text]")
```
Comment (SamHPurp) [inline code/modules/events/sentience.dm]: 😅

File: code/modules/events/sentience.dm [hunk 2]
```diff
@@ -26,9 +26,43 @@
...
+
+/datum/objective/sentFluff/steal
+	explanation_text = "You have the urge to steal as many "
+
+/datum/objective/sentFluff/steal/New()
+	var/things = pick(list("lights","fruits","shoes","bars of soap", "hats", "pants", "organs"))
+	explanation_text += " [things] as you can!"
+
+/datum/objective/sentFluff/fear
+	explanation_text = "In your new found sentience you have learnd"
```
Comment (SamHPurp) [inline code/modules/events/sentience.dm]: `learnd` -> `learned`

File: code/modules/events/sentience.dm (hunk 1 above)
Comment (Aurorablade) [inline code/modules/events/sentience.dm]: i didn't mean it as a refrence..but now that you mention it.....IT DOES make sense though..its not an objective..it is a reason to live...DO NOT LOSE THAT PAIR OF PANTS...

File: code/modules/events/sentience.dm [hunk 3]
```diff
@@ -26,9 +26,43 @@
...
+		var/datum/job/target = pick(jobs)
+		explanation_text += " that [target.title] are evil, avoid these scary things!"
+	else
+		explanation_text += " people are evil, avoid these scary things!"
+
+/datum/objective/sentFluff/toy
+	explanation_text = "You MUST find your special "
+
+/datum/objective/sentFluff/toy/New()
+	var/toy = pick(list("shoe","rubber duck","bike horn","bar of soap", "hat", "par of pants"))
```
Comment (SamHPurp) [inline code/modules/events/sentience.dm]: `par` -> `pair`

**Comment by SamHPurp** [comment]
Eh, I'm not a fan of fluff objectives. Feels limiting, when in truth its supposed to encourage an action.

**Comment by Aurorablade** [comment]
thats what i am trying to do..i am open to add stuff but this is all i could think of.

**Comment by SamHPurp** [comment]
How about once something is sentienced, it sends an admin message suggesting they get given an objective manually. (Which makes me think, did I include an admin log for when something gets sentienced by this?)

Then it makes it more specific to the round at hand, and open if no admin bothers.

**Comment by Aurorablade** [comment]
i will add a log. but the thing about manual objectives..if you see the log spam or other spam we get sometimes it can be EASILY missed. an admin could add more objectives to them, but a fluff objective can be some kind of quirk to make the sentience event not dull.

**Comment by SamHPurp** [comment]
@Aurorablade yeah I have seen it. Hmmm...

**Comment by Fox-McCloud** [comment]
I'd drop the fluff objectives, then this can be merged.

File: code/modules/events/sentience.dm [hunk 4]
```diff
@@ -64,5 +64,5 @@
 	explanation_text = "You MUST find your special "
 
 /datum/objective/sentFluff/toy/New()
-	var/toy = pick(list("shoe","rubber duck","bike horn","bar of soap", "hat", "par of pants"))
+	var/toy = pick(list("shoe","rubber duck","bike horn","bar of soap", "hat", "pair of pants"))
```
Comment (SamHPurp) [inline code/modules/events/sentience.dm]: Inconsistent spacing in the list should be resolved.

**Comment by KasparoVy** [comment]
The mice want covfefe.

**Pull request merged**